.changeset/forty-maps-fly.md

```diff
@@ -0,0 +1,5 @@
+---
+'@itwin/itwinui-react': patch
+---
+
+The [**theme bridge**](https://github.com/iTwin/iTwinUI/wiki/StrataKit-theme-bridge) has been updated to try to match the latest designs from `@stratakit/foundations@0.1.6`.
```

Comment:
Rewording suggestion, because this PR has nothing to do with the version of `@stratakit/foundations`. We can add a separate changeset in #2578 that mentions the version.

```suggestion
The [**theme bridge**](https://github.com/iTwin/iTwinUI/wiki/StrataKit-theme-bridge) has been updated to more closely match the latest StrataKit visuals. The most prominent change is in the background-color of various components.
```